tools/bin/hawq_ctl

```diff
@@ -1041,14 +1041,14 @@ class HawqStop:
             logger.info("Segments %s successfully" % self.stop_action_past)
         return total_return_flag
 
-    def _stop_rps(self):
-        cmd_str = "%s/ranger/bin/rps.sh stop" % (self.GPHOME)
-        result = remote_ssh(cmd_str, self.master_host_name, self.user)
-        return result
 
-    def stop_rps(self, check_ret = True):
```

Comment:
I saw other functions ( e.g. _stop_segment(), _stop_master() ）) in HawqStop class all have '_' prefix, so merge stop_rps() logic to _stop_rps() for unifying them. 
Sounds reasonable?